.github/workflows/cross-compile.yml

```diff
@@ -5,8 +5,6 @@ on:
     branches: [ criu-dev ]
   pull_request:
     branches: [ criu-dev ]
```

Comment:
offtopic: do we need the master branch here?

Reply:
yes, that sounds useful, let met add that.